webapp/src/js/util/customHandlebars.js

```diff
@@ -11,17 +11,27 @@ import resolveJoins from 'panoptes/resolveJoins';
 
 const customHandlebars = ({dataset, tablesById, handlebars}) => {
   let hb = handlebars || promisedHandlebars(Handlebars);
+  hb.registerHelper('concat', function() {
```

Comment:
I believe we already have this installed: https://github.com/helpers/handlebars-helpers#join would it have done?